postman/NCG.postman_collection.json

```
@@ -270,7 +270,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n\t\"hip\": {\n\t\t\"id\": \"10000004\"\n\t}\n}"
+							"raw": "{\n    \"requestId\": \"{{$uuid}}\",\n    \"hip\": {\n        \"id\": \"10000005\"\n    }\n}"
```

```suggestion
							"raw": "{\n    \"requestId\": \"{{$guid}}\",\n    \"hip\": {\n        \"id\": \"10000005\"\n    }\n}"
```
I guess postman does not have $uuid

---

do that everywhere please.

---

sure